### PR TITLE
[pupil-675] & [pupil-674] New pupil browse nav updates

### PR DESCRIPTION
**Pull request by BTitterington**
## Description

Nav changes to make new pupil experience live

**Burger menu** 
 - link to /pupils/years
 - text changed to "home" 
 - no external icon
 
**Footer** 
 - link to /pupils/years
 - Text stayed "Learn online"
 - no external icon 
 - Legal policies reordered see [combined ticket](https://www.notion.so/oaknationalacademy/OWA-Footer-update-Hold-936d01ef23374c2b831c8a82e4f98c53)
 
 **App Header**
  - link to /pupils/years
  - no external icon
  - Aria label - `Pupils (opens in a new tab)` changed to `Pupils browse years`
  - When in section of site with `/pupils` in the url the menu is highlighted
  
 **Pupils Tab**
  - Header changed to to `Learn and revise`
  - Body changed to `We provide free online lessons with videos, quizzes, and more. Whatever the subject, from primary to GCSE, we've got you covered.`
  - Button text changed to `Find a lesson`
  - Button aria-label changed from `Learn with Oak (this will open in a new tab)` to `Find a lesson`
 

## How to test

1. Go to https://deploy-preview-2503--oak-web-application.netlify.thenational.academy
2. See the above changes and check tickets [here](https://notion.so/oaknationalacademy/OWA-Top-Global-Nav-update-34fc5c0637074b1b860c3d8c9c75ae72) and [here](https://www.notion.so/oaknationalacademy/OWA-Footer-update-Hold-936d01ef23374c2b831c8a82e4f98c53) 

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

**Comment by messmike** [comment]
Thanks, @BTitterington, a small tweak to the underline of the pupils menuItem if possible. Can you remove the set width applied to the underlined SVG? E.g:
Screenshot 2024-06-12 at 4 49 16 pm

**Comment by BTitterington** [comment]
@messmike It's weird, I remember trying to fix this and hitting an issue originally (so I put it off). But now when I remove the width everything seems fine, so I've pushed the change. Maybe just check mobile and safari to ensure it still looks good.

**Comment by messmike** [comment]
> 1. https://deploy-preview-2503--oak-web-application.netlify.thenational.academy

All looking good to me! Cheers @BTitterington

File: src/components/SharedComponents/OwaLink/OwaLink.tsx [hunk 1]
```diff
@@ -83,7 +83,6 @@ const StyledNextLink = styled.a<StyleProps>`
 export const ActiveLinkUnderline = styled(Svg)`
   position: absolute;
   height: 8px;
-  width: 72px;
```
Comment (BTitterington) [inline src/components/SharedComponents/OwaLink/OwaLink.tsx:86]: This is only used in the header currently and was approved by UX.

**Review by njblore — APPROVED** [review]

**Comment by aakeshp** [comment]
@BTitterington I presume both design and school support are happy with the copy? Links all work as expected from what I can see.

**Review by RBrownsett — APPROVED** [review]
Approved - testing details are on the ticket

**Pull request merged**